File: sum-consecutives/sum-consecutives.js
function sumConsecutives(s) {
  var t = [s[0]];
  var j=0;
  var i=0;
  while(i<s.length-1){
  if(s[i]==s[i+1]){
  t[j] +=s[i];
  }else{j=j+1;
  t[j]=s[i+1]}  
  i++;}
  return t;
}

File: sum-consecutives/sum-consecutives.py
def sum_consecutives(s):
    t = [s[0]]
    j = 0
    for i in range(len(s)-1):
        if s[i]==s[i+1]:
            t[j] +=s[i]
        else:
            j +=1
            t.append(s[i+1]) 
    return t